packages/neon_framework/lib/neon.dart

```diff
@@ -68,15 +68,15 @@ Future<void> runNeon({
   runApp(
     MultiProvider(
       providers: [
-        NeonProvider<GlobalOptions>.value(value: globalOptions),
-        NeonProvider<AccountsBloc>.value(value: accountsBloc),
-        NeonProvider<FirstLaunchBloc>.value(value: firstLaunchBloc),
-        NeonProvider<NextPushBloc>.value(value: nextPushBloc),
-        Provider<Set<AppImplementation>>(
+        NeonProvider.value(value: globalOptions),
+        NeonProvider.value(value: accountsBloc),
+        NeonProvider.value(value: firstLaunchBloc),
+        NeonProvider.value(value: nextPushBloc),
+        Provider(
           create: (_) => appImplementations,
           dispose: (_, appImplementations) => appImplementations.disposeAll(),
         ),
-        Provider<PackageInfo>.value(value: packageInfo),
+        Provider.value(value: packageInfo),
```

Comment:
Please leave the generics here.
Changing the provided value could otherwise have unwanted side effects down the road.

Reply:
I also added them back to the SelectOption as I feel like one could easily make a mistake there.